app/server.go

```diff
@@ -1721,7 +1721,15 @@ func (s *Server) sendLicenseUpForRenewalEmail(users map[string]*model.User, lice
 }
 
 func (s *Server) doLicenseExpirationCheck() {
-	s.LoadLicense()
```

Comment:
Hey @agnivade do you have any idea why this was added in the first place?

Reply:
This produces a weird effect due to a listener that was added, the listener then sends a websocket to the client with a sanitized license, what happens then is that, on the license page in the console, we have some details missing

Reply:
I don't think I am aware of the context behind this. I think this part is owned by your team so I'll leave it to your best judgement.

Reply:
@AGMETEOR - do you want to bring this back or are you confident this is good to be removed?

Reply:
I am 1/5 removing it